tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -31,6 +29,12 @@ var _ = Describe("odo devfile push command tests", func() {
 		currentWorkingDirectory = helper.Getwd()
 		helper.Chdir(context)
 		os.Setenv("GLOBALODOCONFIG", filepath.Join(context, "config.yaml"))
+
+		// Devfile push requires experimental mode to be set
+		helper.CmdShouldPass("odo", "preference", "set", "Experimental", "true")
+
+		cmpName = helper.RandString(6)
+		helper.CmdShouldPass("odo", "create", "nodejs", "--project", namespace, cmpName)
```

Comment:
I have some concerns about always creating a nodejs component for all of the tests, and baking it into the test suite setup. Could we have some integration tests that do test devfiles from https://github.com/elsony/devfile-registry? (Such as the maven or springboot ones)

Reply:
I will put some springboot tests here and there since it has two component containers.